src/main/core/Manager/FacetManager.php

```diff
@@ -42,4 +43,44 @@ public function deserializeFieldValue($object, $type, $value = null)
 
         return $event->getFormattedValue();
     }
+
+    public function isFieldDisplayed(array $fieldDef, array $allFields, array $data): bool
+    {
+        $condition = $fieldDef['display']['condition'];
+
+        if (!empty($condition)) {
+            $parentField = null;
+
+            foreach ($allFields as $searchedField) {
+                if ($searchedField['id'] === $condition['field']) {
+                    $parentField = $searchedField;
+                }
+            }
+
+            if ($parentField) {
+                $parentValue = ArrayUtils::get($data, 'profile.'.$parentField['id']);
+
+                $displayed = false;
+
+                switch ($condition['comparator']) {
+                    case 'equal':
+                        $displayed = $parentValue === $condition['value'];
+                        break;
+                    case 'different':
+                        $displayed = $parentValue !== $condition['value'];
+                        break;
+                    case 'empty':
+                        $displayed = empty($parentValue);
+                        break;
+                    case 'not_empty':
+                        $displayed = !empty($parentValue);
+                        break;
+                }
+
+                return $displayed;
+            }
+        }
```

Comment:
We had to add `return true;` between those two lines:
```

            return true;
```
https://github.com/konsept-ch/Claroline/pull/63/files

I'll retest with and without this line (I don't remember what issue it was solving) and I'll create a new Issue and/or PR